src/main/java/com/redhat/cloud/custompolicies/app/model/engine/FullTrigger.java

```diff
@@ -90,25 +95,52 @@ private static String generateId() {
     return UUID.randomUUID().toString();
   }
 
+  public void updateFromPolicy(Policy policy) {
+    if (policy.actions == null || policy.actions.isEmpty()) {
+      trigger.actions.clear();
+    }
+    String[] actionsIn = policy.actions.split(";");
+    Set<TriggerAction> newActions = new HashSet<>();
+    for (String actionIn : actionsIn) {
+      if (actionIn.trim().isEmpty()) {
+        continue;
+      }
+      String actionName;
+      if (actionIn.contains(" ")) {
+        actionName = actionIn.substring(0, actionIn.indexOf(' '));
+      }
+      else {
+        actionName = actionIn;
+      }
+      actionName = actionToTriggerActionMap.get(actionName);
+
+      boolean found = false;
+      for (TriggerAction ta : trigger.actions) {
+        String taName = ta.actionPlugin;
+        if (taName.equals(actionName)) {
+          newActions.add(ta);
+          found = true;
```

Comment:
Could you add a `break` after this? 
I suppose that in the future we are also going to check for actionParams (as currently we have no params)